### ISSUE TITLE
Support ref:kozterkep

### ISSUE THREAD

**Issue by thegrasshopper104** [organicmaps/organicmaps#8167]
[Köztérkép](https://www.kozterkep.hu/) is an osm-based database of 43 134 artworks, mostly in Hungary, some in other European countries.
Recently osm-hu decided to move `url` tags of artworks in Hungary to the [ref:kozterkep](https://wiki.openstreetmap.org/wiki/Item:Q22498) tag. _Supporting this would be needed to keep linking to detailed info about all the artworks._ It's basically the same as the url except it's not the official url of the POI.

**Comment by biodranik** [comment]
Is this info also duplicated in Wikipedia?

**Comment by thegrasshopper104** [comment]
> Is this info also duplicated in Wikipedia?

Some of the important ones are, but in my experience the vast majority is not

**Comment by RedAuburn** [comment]
What's the ideal solution? Re-adding a website=* tag?

**Comment by thegrasshopper104** [comment]
> What's the ideal solution? Re-adding a website=* tag?

You mean in OM?
A Köztérkép or more info button would be my ideal solution (like it was with Kayak)

**Comment by RedAuburn** [comment]
> While the website is currently available in Hungarian only, its scope is global. 

maybe only enable it when HU language is selected?

**Comment by thegrasshopper104** [comment]
> > 
> Only 153 tagged so far, all in Hungary.

This is very low priority then.
I'll work on adding more links since this is information that no other map app gives easy access to.
I was under the impression that they auto link everything to OSM points if they use OSM anyways...

**Comment by RedAuburn** [comment]
It's possible that the kotsepek is linked to wikidata, in which case we can get it from there 👍

**Comment by thegrasshopper104** [comment]
So it turns out I misunderstood, the website=kozterkep.hu/* tags weren't removed and won't be for a while, until editors and consumers like OM start supporting the ref: tag

**Comment by infeeeee** [comment]
Old website tags were updated to `ref:kozterkep` recently, so now there are 1200+ tagged artworks. 

JOSM already supports this tag via Tag2Link Addon.

**Comment by biodranik** [comment]
Using a generic website tag would be the easiest for OM, although some conflicts in the editor are expected. Maybe they can be workarounded with a hard-coding.

**Comment by infeeeee** [comment]
> Using a generic website tag would be the easiest for OM

That would be tagging for the renderer: https://wiki.openstreetmap.org/wiki/Tagging_for_the_renderer

---

Tag2Link reads the P8 Formatter url of the tag's OsmWikiDataItem:
- https://wiki.openstreetmap.org/wiki/Item:Q22498
- https://wiki.openstreetmap.org/wiki/Property:P8

More info how they do that:
- https://josm.openstreetmap.de/wiki/Help/Action/Tag2Link
- https://github.com/JOSM/tag2link

OM could do something like that, to automatically read these urls from the osm wiki and we just have to document these kind of tags there, most of them are already documented. With this OM users could get to see a lot more info on a lot of items.

**Comment by biodranik** [comment]
No, that is not tagging for the renderer. The idea is to take the original tag from OSM and store it in OM as website metadata. In this way it will work out of the box everywhere.

One issue is if users decide to edit the website and upload it back to OSM.

Another issue is that there are 80+ existing website tags for these refs.

It can be hacked by adding some special scheme (e.g. kozterkep://12345) to website and a few checks in the code.

Otherwise it requires introducing its own metadata field and patching the code everywhere, including iOS and Android UI.

**Comment by infeeeee** [comment]
Ok, sorry, I misunderstood.

How do you handle for example `contact:facebook` tags where only the id or the name of the page given, not the full url? Isn't that basically the same? ( We even had a vote in the local community if we want to use `contact:kozterkep` instead, bot `ref` was the winner)
